src/bin/server.ts

```diff
@@ -101,6 +101,9 @@ interface Args {
     }
   }
 
+  if (!("emitValueUpdateAfterSetValue" in options))
+    options["emitValueUpdateAfterSetValue"] = true;
```

Comment:
Please use curly braces around if-statements. Don't we have prettier on this repo?

Also, let's print a warning if this is set to `false` that multi-client set ups will not work correctly.